docs/docs/guides/deploy/loadbalancing-example/loadbalancing-example.mdx

```diff
@@ -38,19 +38,19 @@ By executing the commands below, you will download the following files:
 
 ```bash
 # Download the docker compose example configuration. For example:
-wget https://raw.githubusercontent.com/zitadel/zitadel/main/docs/docs/guides/deploye/loadbalancing-example/loadbalancing-example/docker-compose.yaml
+wget https://raw.githubusercontent.com/zitadel/zitadel/main/docs/docs/guides/deploy/loadbalancing-example/docker-compose.yaml
 
 # Download the docker compose example configuration. For example:
```

Comment:
I think this could be named better since this isn't the docker compose configuration. Maybe `Download and adjust the example traefik configuration file` ?

Reply:
Hm, but that file is the compose file 😁 or do I miss the point?

![image](https://user-images.githubusercontent.com/9879976/182147352-78829136-c6b6-45fb-b7e0-98a06d8e05d5.png)

Reply:
I pinned this comment on line 43 which is referring to the link in line 44 which is `wget https://raw.githubusercontent.com/zitadel/zitadel/main/docs/docs/guides/deploy/loadbalancing-example/example-traefik.yaml` 

This is in my opinion the traefik config or am I wrong?

Reply:
Ah sorry 😁  did not get that.

Yes this is the traefik config. That is rendered as a LB example in the docs page https://docs.zitadel.com/docs/guides/deploy/loadbalancing-example

Reply:
```suggestion
# Download the Traefik example configuration. For example:
```

Reply:
Can we ditch the twice "example". IMO "Download the Traefik example configuration." should work fine